projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/profile-tag.ts

```diff
@@ -12,5 +12,23 @@ export const profileTagHelper = {
       return oldAppendChild.call(this, newChild);
     };
   },
+  triggerLoaded() {
+    cy.window().then((win) => {
+      const event = new CustomEvent('profiletag_loaded');
+      win.dispatchEvent(event);
+    });
+  },
+  triggerConsentReferenceLoaded() {
+    cy.window().then((win) => {
+      const event = new CustomEvent('profiletag_consentReferenceLoaded', {
+        detail: { consentReference: profileTagHelper.testCr },
+      });
+      win.dispatchEvent(event);
+    });
+  },
+  waitForCMSComponents(timeout = 7000) {
+    cy.get('cx-profiletag', { timeout });
```

Comment:
I think we can disregard this timeout if we wait for the homepage request using waitForPage. 

The component will be rendered when the cms component is available to the page. 

Also, if we were to use this, we need to update either this or the similar one at the beggining of the test

Reply:
I don't recall what the default timeout is, but I think we dont need 7s: I will remove it.

It has been a few months since I did this, but I recall that homepage request didn't always work because this is just waiting for a rest request, but not necessarily waiting for the component to be available in the dom, and at the end of the day, what we really care about is the DOM element.

I Have updated the wait at the begining of the test to use the one from the profiletaghelper.